src/tenant/initialization.py

```diff
@@ -34,13 +34,21 @@ def load_initial_tenant_data():
 
 
 def create_superuser():
-    # print("Creating default super user of the tenant %s - %s." % (tenant.schema_name, tenant.domain_url))
-    user = User.objects.create_superuser(
-        username=settings.TENANT_DEFAULT_SUPERUSER_USERNAME, 
+    # BYTEDECK ADMIN
+    User.objects.create_superuser(
+        username=settings.TENANT_DEFAULT_ADMIN_USERNAME, 
         email='admin@example.com', 
-        password=settings.TENANT_DEFAULT_SUPERUSER_PASSWORD
+        password=settings.TENANT_DEFAULT_ADMIN_PASSWORD
     )
-    return user
+
+    # OWNER OF THE DECK
+    owner = User.objects.create_superuser(
+        username=settings.TENANT_DEFAULT_OWNER_USERNAME,
+        email='owner@example.com',
+        password=settings.TENANT_DEFAULT_OWNER_PASSWORD,
+    )
+    
+    return owner
```

Comment:
the returned user will be the default teacher of the default blocks, so only one user model should be returned.

Reply:
its also why I mentioned my commit might have fixed #638 assuming you were talking about the default blocks teacher and not the default teacher for newly created blocks

Reply:
i can probably remove that and instead get the owner using User.models.get(username=settings.TENANT_DEFAULT_OWNER_USERNAME) for the blocks_creation

Reply:
Ya that's a good idea, do that.

Reply:
However... that's just the initial username, that username will probably be changed by the owner to something more personal, So we can't search by the username or it will only work for the initially created default blocks, not for blocks created later on by the user?

Reply:
im not exactly sure how to add the owner as the default for blocks that are later created. Since if i go by pk then it will only work if the owner doesnt accidentally delete their user model